bridge/package.json

```diff
@@ -17,7 +17,8 @@
     "deploy": "npx hardhat deploy --show-stack-traces --network",
     "deployIntegrationTest": "npm run deploy rskregtest && npm run deploy development",
     "deployLocalIntegrationTest": "npm run deploy development && npm run deploy mirrorDevelopment",
-    "reDeployLocalIntegrationTest": "ps aux | grep ganache-cli | awk '{print $2}' | xargs kill -9 & npm run ganache & npm run ganache-mirror & npm run deployLocalIntegrationTest"
+    "reDeployLocalIntegrationTest": "ps aux | grep ganache-cli | awk '{print $2}' | xargs kill -9 & npm run ganache & npm run ganache-mirror & npm run deployLocalIntegrationTest",
+    "verify-script": "bash hardhat/script/verify.sh"
```

Comment:
I'd rename to `verifyDeployments`

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/269/commits/4068c20b90d789cd4164cdffe699095cb0a665a4